index.html

```diff
@@ -7,5 +7,7 @@
     <link rel="stylesheet" href="defaults.css" />
     <link rel="stylesheet" href="main.css" />
   </head>
-  <body></body>
+  <body>
+    <header></header>
```

Comment:
Header tag deos not contain html elements, please update before sending a pull request

Reply:
Issue has been fixed, please review